### ISSUE TITLE
It should raise an exception when the installation fails

### ISSUE THREAD

**Issue by HMaker** [ultrafunkamsterdam/undetected-chromedriver#91]
Failed installations pass silently, to test if the installation was successful it's needed to do the following
```python
from undetected_chromedriver import ChromeDriverManager
uc = ChromeDriverManager(executable_path='/path/to/arm/binary/chromedriver', target_version=None)
uc.install()
if not uc.installed:
    raise Exception("installation failed, undetected_chromedriver can't be used ")
```

**Comment by ultrafunkamsterdam** [comment]
@HMaker thanks for your input. However:

1 - first, where do you see the global variable `installed` ? it does not exist.
2 - this code will likely be obsolete in the near future. 

that having said, what's wrong using it just like most others do (that is an assumption, since you are the first ever reporting this, using some code which is obviously not taken from the examples)

i suggest:

``` 
import undetected_chromedriver as uc
driver = uc.Chrome()
```

or 

```
import undetected_chromedriver as uc
uc.install()
from selenium.webdriver import Chrome
driver = Chrome()
```

it works out of the box. and options will initialize with correct values, just as the paths.

**Comment by HMaker** [comment]
Yeah I have not used your library, but some other guy tried to use a
chromedriver ARM binary, by setting the executable_path, to run it on
Raspberry PI but the webdriver was being detected, then I suspected the
binary patching was failed, in fact that happened, but the library did not
make that explicit. I wrote the code above to test the installation, since
the binary patching seems to be a key point, I thought its failure could
not pass silently.

Em Qui, 28 de jan de 2021 22:26, Leon <notifications@github.com> escreveu:

**Comment by ultrafunkamsterdam** [comment]
@HMaker thanks for your explanation. question.. where did you get that arm binary ? made me curious

**Comment by HMaker** [comment]
I did not tested it, I just wrote the code for the other guy to run. It's an arm7l, electron has a binary for it https://github.com/electron/electron/releases/download/v12.0.0-beta.19/chromedriver-v12.0.0-beta.19-linux-armv7l.zip

**Comment by ultrafunkamsterdam** [comment]
Thanks mate
